examples/stable_memory/test/test.ts

```diff
@@ -0,0 +1,78 @@
+import { cleanDeploy, run_tests, Test } from 'azle/test';
+import { createActor } from './dfx_generated/stable_memory';
+
+const stable_memory_canister = createActor('rrkah-fqaaa-aaaaa-aaaaq-cai', {
+    agentOptions: {
+        host: 'http://127.0.0.1:8000'
+    }
+});
+
+const tests: Test[] = [
+    ...cleanDeploy('stable_memory'),
+    {
+        name: 'stable size',
+        test: async () => {
+            const result = await stable_memory_canister.stable_size();
+
+            return {
+                ok: result === 0
+            };
+        }
+    },
+    {
+        name: 'stable64 size',
+        test: async () => {
+            const result = await stable_memory_canister.stable64_size();
+
+            return {
+                ok: result === 0n
+            };
+        }
+    },
+    {
+        name: 'stable grow',
+        test: async () => {
+            const old_size = await stable_memory_canister.stable_size();
+            const new_pages = 5;
+            const result = await stable_memory_canister.stable_grow(new_pages);
+            const new_size = await stable_memory_canister.stable_size();
+
+            return {
+                ok:
+                    'ok' in result &&
+                    result.ok === old_size &&
+                    new_pages + old_size === new_size
+            };
```

Comment:
@bdemann Just in case you hadn't see it before, you could explicitly handle the err case of your result first:

```ts
import { ok } from 'azle'

if(!ok(result)) {
  return { err: result.err };
}
```
Then afterwards you'd still have to check `result.ok === old_size && new_pages + old_size === new_size`, but the advantage would be that if there was an error, the testing framework would print out the err message, vs with that you have the test would just fail, but wouldn't show what the error message was.

Reply:
My results are ```StableGrowResults``` that don't seem to work with ```ok```
The best I was able to come up with is 
```
            if ('err' in result) {
                return {
                    err: Object.getOwnPropertyNames(result.err)[0]
                };
            }
```
Which seems a little clunky to me. What are you thoughts? Or is there a way to make StableGrowResults work with ok?